src/sql/strong-oracle/create.sql

```diff
@@ -0,0 +1,332 @@
+BEGIN
```

Comment:
> General standard is that all index script should start with IDX_

@nararama These names were @scratch28's suggestions, so please discuss with him. We have a separate task for renaming, so we can save those changes for a future pull request.